sonic_platform_base/sonic_eeprom/eeprom_base.py

```diff
@@ -220,7 +221,8 @@ def open_eeprom(self):
         except Exception:
             pass
         self.cache_update_needed = using_eeprom
-        return io.open(eeprom_file, "rb")
+        self.eeprom_file_handle = io.open(eeprom_file, "rb")
+        return using_eeprom
```

Comment:
Unfortunately, changing the return value will break all existing code which is already using this library. These changes will need to be implemented incrementally. This PR should store the EEPROM file handle in a member variable as you have done, but it should not break existing functionality. All methods which currently take the EEPROM file handle as a parameter should be modified such that the parameter defaults to `None`, and if the handle is `None`, then the function should use `self.eeprom_file_handle`. This will allow vendors to update their existing code to stop passing the handle to methods. Once all vendors have transitioned, then we can change this return value to a boolean.

Reply:
@jleveque: Does it mean that we have to store the file handle again in a temporary variable in the `read_eeprom_bytes` function but use the `eeprom_file_handle` after the function call to update the `eeprom_file` file path?

Reply:
`read_eeprom_bytes` does not take the EEPROM file handle as a parameter. However, other methods do.